recipes/pypromice/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pypromice" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pypromice-{{ version }}.tar.gz
+  sha256: a4872db097cc8814c31bf14c5a40cada0fa345996126ae865c2402db260e4ca8
+
+build:
+  skip: true  # [py<38]
```

Comment:
Looks like this package has scripts and cannot be noarch. Can you open an issue upstream and ask them to move to entry_points? Scripts are deprecated and cause all sorts of trouble for downstream packagers.

Reply:
You can't do that here now. Upstream must convert those scripts to entry_points. See https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#entry-points

For now, we can merge this without noarch but it is highly recommend to change that I'm the future.

Reply:
But if it cannot be `noarch` then we will not have cross-platform compatibility, right? 

If this is the case, then we would like to make the upstream changes and move the scripts to `entry_points` before merging.

As pypromice maintainers, we can start moving the scripts as soon as possible.

Reply:
That's great. Usually folks are eager to get packages in without waiting for upstream changes and new releases.